docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Summary of Changes
+
+Replace this text with a high-level summary of the changes included in this PR.
+
+# Related Issues
+
+Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.
+
+# Pull Request Checklist
+
+This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
+
+If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.
+
+- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
+- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
+- [ ] Run prettier - Instructions [here](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
```

Comment:
We should indicate that you should run npm run style-format instead of linking externally, which means we aren't dependent on external processes or documentation.

Reply:
I don't see the "format" script. There's a style-format and style-check, but I don't see just format.

Reply:
You're right! I had AFJ up and was looking in the wrong file.